### PR TITLE
fix(files): Fix file layout resetting renaming action

### LINKED ISSUE #1374: bug(files): creating folder doesnt do anything
STR:
- cargo run
- files
- click on create folder
- doesn't do anything

macOS, m1

### LINKED ISSUE #1384: bug(files): rename files doesn't work
STR:
- go to files
- right click
- rename file
- doesn't do anything

Captura de ecrã 2023-10-23, às 22 06 46

macOS, M1

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Fixes file layout having an onclick that immediately resetted actions such as renamine.
  - I guess it was put there to auto rename if e.g. the user clicks of the folder but even without it it already does it so it might be redundant. @lgmarchi asking you to verify since you created the pr for it

### Which issue(s) this PR fixes 🔨

- Resolve #1374
- Resolve #1384

### CODE REVIEW

File: ui/src/layouts/storage/files_layout/mod.rs [hunk 1]
```diff
@@ -166,7 +163,9 @@ pub fn FilesLayout(cx: Scope<'_>) -> Element<'_> {
                                         }
                                     )),
                                     onpress: move |_| {
+                                        log::debug!("click {} ", upload_file_controller.files_been_uploaded.read());
                                         if !*upload_file_controller.files_been_uploaded.read() {
+                                            log::debug!("r ");
```
Comment (dariusc93) [inline ui/src/layouts/storage/files_layout/mod.rs:168]: What does 'r' symbolize here while logging?

**Review by sdwoodbury — APPROVED** [review]
requesting the debug statement get changed from "r" to something else, like "finished renaming". otherwise looks good.

**Comment by stavares843** [comment]
this is already fixed in current dev with hogan's PR [here](https://github.com/Satellite-im/Uplink/pull/1393)

do we want this PR? cc @tooshel @WanderingHogan

**Comment by Flemmli97** [comment]
ye. its fixed in dev already so we should close this pr

**Comment by stavares843** [comment]
thanks @Flemmli97 🔨

**Pull request closed without merging**